examples/PoCTutorial/src/esdl_generator.py

```diff
@@ -0,0 +1,409 @@
+    demand_1_coord = [52.00107, 4.37353]
+    
+    # Demand 2 parameters
+    demand_2_power = 15e6       # W
+    demand_2_inv_cost = 1e6     # eur/MW
+    demand_2_inst_cost = 1e6    # eur
+    area_2_coord = [[51.99817258139733, 4.3692970275878915], [51.99932194116153, 4.373695850372315],
+                    [51.99534529503086, 4.376678466796876], [51.99624370444482, 4.38081979751587],
+                    [51.99490929572965, 4.381613731384278], [51.99272923553559, 4.372708797454835]]
+    demand_2_coord = [51.99653, 4.3731]
+
+    # Demand 3 parameters
+    demand_3_power = 15e6       # W
+    demand_3_inv_cost = 1e6     # eur/MW
+    demand_3_inst_cost = 1e6    # eur
+    area_3_coord = [[51.992279997820766, 4.372987747192384], [51.994235470276436, 4.382214546203614],
+                    [51.989333415199866, 4.38581943511963], [51.98688879367896, 4.37633514404297]]
+    demand_3_coord = [51.99074, 4.3791]
+
+    # Source 1 parameters
+    source_1_power = 50e6       # W
+    source_1_inv_cost = 1.5e6   # eur/MW
+    source_1_inst_cost = 10e6   # eur
+    source_1_varop_cost = 100.0   # eur/MWh
+    source_1_fixop_cost = 6e5   # eur/MW
+    prod_1_coord = [52.00483, 4.37272]
+
+    # Source 2 parameters
+    source_2_power = 50e6       # W
+    source_2_inv_cost = 2e6     # eur/MW
+    source_2_inst_cost = 20e6   # eur
+    source_2_varop_cost = 200.0   # eur/MWh
+    source_2_fixop_cost = 9e5   # eur/MW
+    prod_2_coord = [51.99543, 4.36457]
+
+    # Joints
+    joint_1_coord = [52.00098, 4.36702]
+    joint_2_coord = [51.99596, 4.36951]
+    joint_1ret_coord = [52.00033, 4.36646]
+    joint_2ret_coord = [51.99537, 4.36895]
+
+    # Pipes
+    pipe_1_elbow_coord = [52.00353, 4.36593]
+    pipe_1ret_elbow_coord = [52.00363, 4.36492]
+    pipe_3_elbow_coord = [51.9898, 4.37191]
+    pipe_3ret_elbow_coord = [51.98922, 4.3717]
+
+    ######################################################
+    ##                  PROCESSING                      ##
+    ######################################################
+
+    esh = EnergySystemHandler()
+    energy_system = esh.create_empty_energy_system(name = 'Generated PoC tutoral')
+    main_area = energy_system.instance[0].area  # Logical high level area.
+
+    # Create heat carriers
+    heat_commodity_primary = esdl.HeatCommodity(id = str(uuid4()), name = 'HeatCommodityPrimary', supplyTemperature = temp_supply)
+    heat_commodity_ret = esdl.HeatCommodity(id = str(uuid4()), name = 'HeatCommodityRet', returnTemperature = temp_ret)
+    heat_carrs = esdl.Carriers(id = str(uuid4()))
+    heat_carrs.carrier.append(heat_commodity_primary)
+    heat_carrs.carrier.append(heat_commodity_ret)
+    for carrier in heat_carrs.carrier:
+        if carrier.name == 'HeatCommodityPrimary':
+            id_primary_carr = carrier.id
+        elif carrier.name == 'HeatCommodityRet':
+            id_ret_carr = carrier.id
```

Comment:
This loop is somewhat unnecessary, you could get the id of the primary carrier just by
id_primary_carrier = heat_commodity_primary.id
And similarly for the secondary carrier.